core/keyboard_nav/cursor.js

```diff
@@ -29,39 +30,39 @@ goog.provide('Blockly.Cursor');
 
 /**
  * Class for a cursor.
- * @param {boolean=} opt_marker True if the cursor is a marker. A marker is used
- *     to save a location and is an immovable cursor. False or undefined if the
- *     cursor is not a marker.
+ * A cursor controls how a user navigates the blockly ast.
  * @constructor
  */
-Blockly.Cursor = function(opt_marker) {
+Blockly.Cursor = function() {
   /*
    * The current location of the cursor.
-   * @type {Blockly.Field|Blockly.Connection|Blockly.Block}
+   * @type {Blockly.ASTNode}
    * @private
    */
   this.curNode_ = null;
+};
 
-  /**
-   * Whether or not the cursor is a marker.
-   * @type {boolean} True if the cursor is a marker. False otherwise.
-   * @private
-   */
-  this.isMarker_ = !!opt_marker;
+/**
+ * The object in charge of drawing the visual representation of the current node.
+ * @type {Blockly.CursorSvg}
+ * @private
+ */
+Blockly.Cursor.prototype.drawer_ = null;
```

Comment:
Can you move this to the constructor. According to the Google Style guide all properties should be defined in the constructor (there's a browser performance reason for that).